src/main/java/com/box/sdk/BoxJSONObject.java

```diff
@@ -82,11 +83,7 @@ public String getPendingChanges() {
      * @return a JSON string containing the pending changes.
      */
     public JsonObject getPendingChangesAsJsonObject() {
-        JsonObject jsonObject = this.getPendingJSONObject();
-        if (jsonObject == null) {
-            return null;
-        }
```

Comment:
This is strange.  Before we remove it, do we understand why this was done, in the first place?
I found [this comment](https://github.com/box/box-java-sdk/pull/461/files?authenticity_token=j3CTEyix06FPuFrpS5IKflKclTzDG9PI4bx1NgwETkpfqlmKM%2BOFzIR8IHJ1WIQbtfgIwTO82LsuUkbmxc7nJw%3D%3D&file-filters%5B%5D=.java#r135368683), which doesn't actually help. lol
Any other thoughts?
Is this covered by tests?

Reply:
I think it was a mistake or a change in the solution. Check the DOCS comment
`return a JSON string containing the pending changes`
If the method would return a String then this approach would make sense. But it looks that idea changed and instead of returning String a JsonObject was returned. We do not need null checks here.